PsfLauncher/ErrorInformation.h

```diff
@@ -0,0 +1,28 @@
+#pragma once
+#include <string>
+#include <windows.h>
+#include <iostream>
+#include "utilities.h"
+class ErrorInformation
+{
+public:
+    ErrorInformation();
+    ErrorInformation(std::wstring customMessage, DWORD errorNumber);
+    ErrorInformation(std::wstring customMessage, DWORD errorNumber, std::wstring exeName);
+    ~ErrorInformation();
+    ErrorInformation(const ErrorInformation &toCopy);
+    ErrorInformation& operator=(const ErrorInformation &toAssign);
+
+    std::wstring Print();
+    bool IsThereAnError();
+    DWORD GetErrorNumber();
+    void AddExeName(std::wstring exeNameToAdd);
+
+private:
+    std::wostringstream customMessage;
+    DWORD errorNumber;
+    std::wstring exeName;
+    std::wstring errorMessage;
+    bool isThereAnError;
+};
+

```

Comment:
this whole class could be defined inline (delete the cpp)

Reply:
Why should I define this inline, and, what is the reason for doing so?

Also, what do you mean by "Inline" defining all the logic in the .h file?

Reply:
yes, all in the .h:
- more efficient (inline methods)
- fewer source files
- easier to read/maintain

Reply:
Done.